core/esmf-aspect-model-aas-generator/src/main/java/org/eclipse/esmf/aspectmodel/aas/Context.java

```diff
@@ -12,26 +12,68 @@
  */
 package org.eclipse.esmf.aspectmodel.aas;
 
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.Objects;
 import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.stream.Collectors;
 
+import org.eclipse.digitaltwin.aas4j.v3.model.AssetKind;
 import org.eclipse.digitaltwin.aas4j.v3.model.ConceptDescription;
 import org.eclipse.digitaltwin.aas4j.v3.model.Environment;
+import org.eclipse.digitaltwin.aas4j.v3.model.ModellingKind;
 import org.eclipse.digitaltwin.aas4j.v3.model.Submodel;
 import org.eclipse.digitaltwin.aas4j.v3.model.SubmodelElement;
 
+import com.fasterxml.jackson.databind.JsonNode;
 import org.eclipse.esmf.metamodel.Property;
 
+
+/**
+ * Contains and tracks the context while the {@link AspectModelAASVisitor} traverses the metamodel.
+ */
 public class Context {
 
    Environment environment;
```

Comment:
Make all those fields that can be `final`

Reply:
Done.